docs/docs/stories.mdx

```diff
@@ -18,23 +18,23 @@ Here's an example of a dialogue in the Rasa story format:
 stories:
 - story: collect restaurant booking info  # name of the story - just for debugging
   steps:
-  - intent: greet                         # user message with no entities
-  - action: utter_ask_howcanhelp
-  - intent: inform                        # user message with no entities
+  - intent: greet                          # user message with no entities
+  - action: utter_ask_howcanhelp           # action that the bot should execute
+  - intent: inform                         # user message with entities
     entities:
     - location: "rome"
     - price: "cheap"
-  - action: utter_on_it                  # action that the bot should execute
+  - bot: On it                             # actual text that bot can output
   - action: utter_ask_cuisine
-  - intent: inform
-    entities:
-    - cuisine: "spanish"
+  - user: I would like [spanish](cuisine). # actual text that user input
```

Comment:
considering this an experimental feature I'm wondering if it's worth adding this to the first example on this page here. In my opinion the introduction to the format should be clear and concise while covering the essential use cases. We can add formats for further usages in further sections and making it clear that this is an experimental feature.

Reply:
I suggest having an additional section with all end-to-end things (similar as we have it for `checkpoints` or `OR statements`) instead of mixing it.

Reply:
reverted this example

Reply:
I don't think additional section will make it better, because you can mix them. I think current structure in `User Messages` and `Actions` is better